### PR TITLE
Escape arguments before appending them to the url to be queried.

### PR DESCRIPTION
**Pull request by georgestephanis**

### CODE REVIEW

File: modules/comment-likes/comment-like-count.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ jQuery( document ).ready( function( $ ) {
 	function fetchCounts() {
 		var batchRequest = {
 			path: '/batch',
-			data: 'urls[]=' + APIqueue.join( '&urls[]=' ),
+			data: 'urls[]=' + APIqueue.map( 'encodeURIComponent' ).join( '&urls[]=' ),
```
Comment (zinigor) [inline modules/comment-likes/comment-like-count.js]: Sorry, but you need `map( encodeURIComponent )` here instead of `map( 'encodeURIComponent' )`.

Comment (vindl) [inline modules/comment-likes/comment-like-count.js]: Agreed, otherwise we'll run into: `Uncaught TypeError: encodeURIComponent is not a function`.

Comment (vindl) [inline modules/comment-likes/comment-like-count.js]: Updated in f642d865190c72e5eda91fec9fffd8ae0760fbec.

**Comment by eliorivero** [comment]
Cherry picked to 5.1 in 1168c5097e8a4bf14c93bda4318c22ad8bad063e

**Pull request merged**